rotkehlchen/chain/ethereum/adex/adex.py

```diff
@@ -1005,7 +1021,7 @@ def get_events_history(
         """Get the staking history events of the addresses in the AdEx protocol.
 
         May raise:
-        - RemoteError: when there is a problem querying the subgraph.
+        - RemoteError: when there is a problem either querying the subgraph.
```

Comment:
what did you add this here for? Either querying the subgraph ..... or ... ?

Reply:
Forgot to update this docstring. Now, an error querying the subgraph or deserializing the response (events) will raise RemoteError, that will bubble up